rules/linux/command_and_control_tunneling_via_earthworm.toml

```diff
@@ -20,7 +20,12 @@ license = "Elastic License v2"
 name = "Potential Protocol Tunneling via EarthWorm"
 note = """## Setup
 
-If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
+If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2,
+events will not define `event.ingested` and default fallback for EQL rules was not added until version 8.2.
+Hence for this rule to work effectively, users will need to add a custom injest pipeline to populate
+`event.ingested` to @timestamp.
+
+For more details to add a custom injest pipeline refer - https://www.elastic.co/guide/en/fleet/current/data-streams-pipeline-tutorial.html
```

Comment:
Lets mention what level of logging has to be turned on for Elastic defend to log the events needed for the rule.

Refer users to docs about setup of auditbeat: https://www.elastic.co/guide/en/beats/auditbeat/current/auditbeat-overview.html.
refer users to docs about Elastic Defend: https://www.elastic.co/guide/en/security/current/install-endpoint.html